### PR TITLE
Readme.md: Refine the Readme.

### PR DESCRIPTION
**Pull request by ysun**
Remove the TO-DO list which has been completed after last release. Add usage of new test runner runtests.

Signed-off-by: Yi Sun <yi.sun@intel.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,13 +25,47 @@ cd lkvs
...
+
+Normally, there're one or more executable binaries or scirpts in a sub-project, once it is compiled successfully. The easiest way is to run them directly.
+
+## Case runner
+
+**runtests** is a straightforward test runner that can execute a single or a set of test cases and redirect the log.
+
+There are 2 ways to pass test to **runtests**:
+  1. `-c`: Pass test cmdline.
+  2. `-f`: Record the cmdline in a `tests` file, then pass the file to the runtests, which can contain multiple tests.
```
Comment (xupengfe) [inline README.md:56]: "
`

> 2. `-f`: Record the cmdline in a `tests|tests-client|tests-server` file, all these tests files are in each component folder, please fill the tests files after -f.

`

Other thing is good to me.

Thanks!

**Review by xupengfe — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (hanning0511) [reply README.md:56]: `tests*` should be better, cause we only need the file name starts with **tests**. Asterisk can be replaced by any resonable strings.

Comment (xupengfe) [reply README.md:56]: Yes, tests* is better for me.

> -f: Record the cmdline in a tests* file, all these tests* file are in each component folder, please fill the tests* files after -f.

Thanks!

**Review by xupengfe — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -27,13 +25,47 @@ cd lkvs
...
+
+Normally, there're one or more executable binaries or scirpts in a sub-project, once it is compiled successfully. The easiest way is to run them directly.
+
+## Case runner
+
+**runtests** is a straightforward test runner that can execute a single or a set of test cases and redirect the log.
+
+There are 2 ways to pass test to **runtests**:
+  1. `-c`: Pass test cmdline.
+  2. `-f`: Record the cmdline in a tests\* file, all these tests\* file are in each component folder, please fill the tests\* files after -f.
```
Comment (xupengfe) [inline README.md:56]: all these tests\* file (file -> files) are in each component folder
My bad.

**Review by xupengfe — CHANGES_REQUESTED** [review]

**Review by xupengfe — APPROVED** [review]

**Pull request merged**